package.json

```diff
@@ -14,7 +14,7 @@
   "devDependencies": {
     "babel-core": "^6.26.0",
     "babel-jest": "^21.2.0",
-    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-env": "^1.6.1",
```

Comment:
are we sure about using this? the project seems to have moved? https://github.com/babel/babel-preset-env